common/common-ui/src/main/res/values/design-experiments-theming.xml

```diff
@@ -17,11 +17,15 @@
 <resources>
 
     <style name="Theme.DuckDuckGo.Dark.Experiment.Warm" parent="Theme.DuckDuckGo.Dark">
-
+        <item name="preferredStatusBarColor">@color/warm_gray90</item>
```

Comment:
this is where all the color change. the experimental ui has a different status bar color, which matches the toolbar.